components/x-article-save-button/src/ArticleSaveButton.jsx

```diff
@@ -0,0 +1,53 @@
+import { h } from '@financial-times/x-engine';
+import articleSaveStyles from './ArticleSaveButton.scss';
+import classNames from 'classnames';
+
+export const ArticleSaveButton = props => {
+	const getLabel = props => {
+		if (props.saved) {
+			return 'Saved to myFT';
+		}
+
+		return props.contentTitle ? `Save ${props.contentTitle} to myFT for later` : 'Save this article to myFT for later';
+	};
+
+	return (
+		<form
+			className={classNames(articleSaveStyles.root)}
+			action={`/myft/save/${props.contentId}`}
+			method="GET"
+			data-content-id={props.contentId}
+			onSubmit={event => {
+				event.preventDefault();
+				const detail = {
+					action: props.saved ? 'remove' : 'add',
+					actorType: 'user',
+					actorId: null, // myft client sets to user id from session
```

Comment:
Does this need specifying at all? From my point of view this doesn't mean anything and is a bit confusing.

Reply:
No it doesn't. As the code that actually performs the save action lives outside of this component, there's no need to even mention this here. I'll remove it.